packages/editor/src/components/editor-help/help-detail-navigation-screen.native.js

```diff
@@ -1,21 +1,21 @@
 /**
  * External dependencies
  */
-import { View, Text } from 'react-native';
-import { useNavigation } from '@react-navigation/native';
+import { View } from 'react-native';
 
 /**
  * WordPress dependencies
  */
 import { BottomSheet } from '@wordpress/components';
+import { useNavigation } from '@react-navigation/native';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import styles from './style.scss';
 
-const HelpDetailNavigationScreen = ( { name } ) => {
+const HelpDetailNavigationScreen = ( { name, view } ) => {
```

Comment:
I wondered if there might be a more descriptive name than `view` for this prop. Possibly `content` or `helpContent`? WDYT? 

Happy to leave this as-is if you disagree. 🙇🏻

Reply:
I like it! Updated in af2828549537bf99702695431ab8c8435788952b